ci-operator/config/openshift/assisted-service/openshift-assisted-service-master.yaml

```diff
@@ -46,6 +50,12 @@ tests:
       SOURCE_IMAGE_REF: agent-installer-api-server
     test:
     - ref: assisted-baremetal-images-publish
+- as: e2e-agent-compact
+  optional: true
+  run_if_changed: ^(Dockerfile\.assisted-service\.ocp)|(cmd/agentbasedinstaller)
+  steps:
+    cluster_profile: equinix-ocp-metal
```

Comment:
There are 2  possible accounts this could go on, other jobs on the repository are using "packet-assisted" while ipi jobs use the "equinix-ocp-metal" profile, I guess it depend which group this job is benefiting. If this job will be using equinix-ocp-metal can you add "cluster: build05" so that this job will have access to our internal hardware and use it if any is available.

Reply:
> There are 2 possible accounts this could go on, other jobs on the repository are using "packet-assisted" while ipi jobs use the "equinix-ocp-metal" profile, I guess it depend which group this job is benefiting.

This is really for the agent-based installer team, so I think equinix-ocp-metal is right, @celebdor can confirm.

> If this job will be using equinix-ocp-metal can you add "cluster: build05" so that this job will have access to our internal hardware and use it if any is available.

Can do. Presumably we want this for https://github.com/openshift/release/blob/master/ci-operator/config/openshift/installer/openshift-installer-master.yaml#L506-L511 as well? I can follow up with a separate PR.

Reply:
#32368